PageGetter.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * This is the Dokuwiki export made by Dominik Brader for FINDOLOGIC.
+ *
+ * If any bugs occur, please submit a new issue
+ * @see https://github.com/findologic/dokuwiki-plugin-findologic-xml-export/issues/new
+ * @author Dominik Brader <support@findologic.com>
+ */
+
+class PageGetter
+{
+    /**
+     * @return array all pages that do not have a title set.
+     */
+    static function getPagesWithoutTitle()
+    {
+        $indexer = new Doku_Indexer();
+        $allPages = $indexer->getPages();
+
+        foreach ($allPages as $page) {
+            if (p_get_metadata($page)['description']) { // Only get pages with content
```

Comment:
What about pages without description and without title? The comment should not only describe that you skip them, but especially also why you skip them.

Reply:
Instead of the `foreach` loop you could use `array_filter` with a nice closure, but it's not necessarily better.